changelog/21460.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+storage/raft: Add experimental support for raft-wal, a new backend engine for integrated storage.
```

Comment:
@raskchanky next time please use the correct new feature formatting for new features in the changelog.

Reply:
Thanks for the reminder. I did correct this in a subsequent PR.

Reply:
This seems like a good candidate for a new CI check, so that we don't have to rely on humans remembering to always do the right thing in several different scenarios.

Reply:
@raskchanky I've added an agenda item to discuss new requirements for the changelog checking tooling.